### DISCUSSION TITLE
(starship::utils): Executing command "/usr/bin/git" timed out

### DISCUSSION THREAD

**Discussion by mrturcot** [starship/starship#4181] [Q&A] [answered]
Hello, sometimes when I enter a git directory I get this message.

```
[WARN] - (starship::utils): Executing command "/usr/bin/git" timed out.                                                                                       
[WARN] - (starship::utils): You can set command_timeout in your config to a higher value to allow longer-running commands to keep executing.
```

So I understand from here https://starship.rs/config/#prompt

`command_timeout | 500 | Timeout for commands executed by starship (in milliseconds).`

So I set "command_timeout = 1000" in "~/.config/starship.toml" but then I get this message.

`[WARN] - (starship::config): Failed to load config value: Error in 'CmdDuration' at 'command_timeout': Unknown key`

What am I doing wrong here??

starship.toml - https://hastebin.skyra.pw/ihubuhupos.bash
.zshrc - https://hastebin.skyra.pw/rutexejaka.ini

**Comment by davidkna** [comment, marked as answer]
Top-level configuration keys, such as `command_timeout`,  need to be at the start of the file, before any subsection. You can also edit the file with `starship config command_timeout 1000`.

**Reply by AlbertSnows** [reply]
small addendum, if you use any starship presets that overwrite your config, then this command needs to go after that in whichever runtime config file you're using.
